client/components/update-post-status/index.js

```diff
@@ -48,7 +49,15 @@ const getStrings = once( translate => ( {
 	},
 } ) );
 
-const enhance = flow( localize, connect( null, { recordGoogleEvent } ) );
+const enhance = flow(
+	localize,
+	connect(
+		( state, props ) => ( {
+			site: getSite( state, get( props, 'post.site_ID' ) ),
```

Comment:
Is there a reason why we don't use `getSelectedSite` here?
I mean, are there cases when the post we're updating is not one from the currently selected site?

Reply:
That's a nice question, I was not very familiar with the codebase and all the functionality and given that the post contains the siteId at the time I preferred not to risk. But right now I think it's safe to use getSelectedSite I will make a try :)